docs/topics/using/edgectl/service-preview-install.md

```diff
@@ -2,42 +2,231 @@
 
 Service Preview is installed as an addon to the Ambassador Edge Stack.
 
-There are two main mechanisms for installing Service Preview
-
-- [`edgectl install`](#install-with-edgectl) will boot strap your cluster with the Ambassador Edge Stack and Service Preview and only works when doing a fresh install of Ambassador.
-
-- [`YAML Install`](#install-with-yaml) will work with an existing installation of Ambassador Edge Stack.
-
 ## Prerequisites
 
 - [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) access to a Kubernetes cluster
-- The Ambassador Edge Stack [Installed](../../../../tutorials/getting-started/)
 - [`edgectl`](../edge-control#installing-edge-control) client
 
-## Install with Edgectl
+## Install
+
+There are three method for installing Service Preview.
+
+### [<img class="Ambassador's OpenSource Blackbird" src="../../../images/features-page-bird.svg"/> Install with Edgectl](#install-with-edgectl)
+
+If you are installing Service Preview and Ambassador Edge Stack for the first time, `edgectl` will bootstrap your cluster with the Ambassador Edge Stack and Service Preview automatically.
```

Comment:
`Service Preview and Ambassador Edge Stack` then `Ambassador Edge Stack and Service Preview` is a bit redundant. Can we rephrase this?